tox.ini

```diff
@@ -1,104 +1,28 @@
 [tox]
-envlist = py25, py26, py27, pypy, py_django13, py_django12, postgres, mysql, postgres_tz, mysql_tz, sqlite_tz, mysql_tz_nopytz, postgres_tz_nopytz, sqlite_tz_nopytz
+envlist =
+    py{35,27}-dj{dev,111,110,19,18}-{sqlite,postgres,mysql,postgres_tz,mysql_tz,sqlite_tz,mysql_tz_nopytz,postgres_tz_nopytz,sqlite_tz_nopytz}
 
 [testenv]
-deps=
-    python-dateutil == 1.5
-    django == 1.4
-
-commands=
-    django-admin.py test qsstats --settings=test_settings.sqlite []
-
-
-
-[testenv:py_django13]
-deps=
-    python-dateutil == 1.5
-    django == 1.3.1
-
-[testenv:py_django12]
-deps=
-    python-dateutil == 1.5
-    django == 1.2.7
-
-
-
-[testenv:postgres]
-deps=
-    python-dateutil == 1.5
-    django == 1.4
-    psycopg2
-
-commands=
-    django-admin.py test qsstats --settings=test_settings.postgres []
-
-[testenv:postgres_tz]
-deps=
-    python-dateutil == 1.5
-    django == 1.4
-    psycopg2
-    pytz
-
-commands=
-    django-admin.py test qsstats --settings=test_settings.postgres_tz []
-
-[testenv:postgres_tz_nopytz]
-deps=
-    python-dateutil == 1.5
-    django == 1.4
-    psycopg2
-
-commands=
-    django-admin.py test qsstats --settings=test_settings.postgres_tz []
-
-
-
-
-[testenv:mysql]
-deps=
-    python-dateutil == 1.5
-    django == 1.4
-    mysql-python
-
-commands=
-    django-admin.py test qsstats --settings=test_settings.mysql []
-
-[testenv:mysql_tz]
-deps=
-    python-dateutil == 1.5
-    django == 1.4
-    pytz
-    mysql-python
-
-commands=
-    django-admin.py test qsstats --settings=test_settings.mysql_tz []
-
-[testenv:mysql_tz_nopytz]
-deps=
-    python-dateutil == 1.5
-    django == 1.4
-    mysql-python
-
-commands=
-    django-admin.py test qsstats --settings=test_settings.mysql_tz []
-
-
-
-
-[testenv:sqlite_tz]
-deps=
-    python-dateutil == 1.5
-    django == 1.4
-    pytz
-
-commands=
-    django-admin.py test qsstats --settings=test_settings.sqlite_tz []
-
-[testenv:sqlite_tz_nopytz]
-deps=
-    python-dateutil == 1.5
-    django == 1.4
-
-commands=
-    django-admin.py test qsstats --settings=test_settings.sqlite_tz []
-
+setenv =
+    PYTHONPATH = {toxinidir}
+basepython =
+    py27: python2.7
+    py35: python3.5
+deps=
+    python-dateutil == 2.6.1
```

Comment:
in README it is stated date < 2.0 is required; could you please fix README as well?